stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/compose/messages/MessageListHeader.kt

```diff
@@ -61,6 +63,7 @@ private object MessageListHeaderUsageSnippet {
                             messageMode = messageMode,
                             onBackPressed = { },
                             onHeaderActionClick = { },
```

Comment:
Seeing how now every individual content area is clickable, do you think it might be prudent to now rename this to something like `onHeaderTitleClick`.

Reply:
I agree with you. `onHeaderActionClick` sounds ambiguous sometimes.